wiki/Guides/Getting_Your_Map_Modded/en.md

```diff
@@ -1,38 +1,43 @@
-- Visiting a [special map listing](https://old.ppy.sh/p/beatmaplist&r=8) on the main site will list the 20 most recent requests plus a few extras.
+## Modding queues
 
-If you post more than two map links in a short period of time, Bancho will ignore them.
+Posting your beatmap in modding queues is the most effective, and most widely used, way of finding people to mod your beatmaps. Modding queues are threads posted in the [Modding Queues](https://osu.ppy.sh/community/forums/60) forum that allow the creator of the thread to find beatmaps to mod, and for others to easily find modders to mod their beatmaps. Using the Modding Queues forum to find mods is straightforward and effective—just follow the steps below to get started:
 
-It's not the most effective method, but it doesn't hurt to try (just don't count on it).
+1. Open the [Modding Queues](https://osu.ppy.sh/community/forums/60) page in your web browser.
+2. Have a look through the list of queues and find one that accepts beatmaps from the game mode you have mapped, and make sure it is marked as "open" in the title or post.
+3. Read the queue's rules. If you or your beatmap does not meet the requirements listed there, then unfortunately you will have to find another queue to post in.
+4. Read the modder's preferences listed in the queue (if any), as it may save both you and the creator of the queue a lot of time.
+5. Click the `go to last post` button at the bottom of the screen. This is represented by the double right arrow icon.
+6. Type what sort of request you would like, along with a link to your beatmap and any other information the modder has asked you to provide, into the `New reply` box.
+7. Click `Post` when you are finished.
 
-## Modding queues
+It is generally acceptable to post in a few modding queues at a time, so feel free to repeat this process if necessary, or if you want to boost your chances of receiving a mod. Keep in mind, however, that just because you have posted in a modding queue, it does not mean the modder is obligated to mod your beatmap; they will mod it if they want to.
+
+It is worth noting that you should avoid posting in Beatmap Nominators' queues until your mapping is at a near rankable level (unless they have specified they are looking for newer mappers), as they will likely always ignore your requests otherwise. You are much better off finding normal modders to mod your beatmaps when you are still learning how to beatmap.
+
+## #modreqs
 
-The [Modding Queues](https://osu.ppy.sh/community/forums/60) section of the forum contains Modding Queues (hard to comprehend, we know). Modding Queues are places where you post maps, and the creator of the thread may eventually mod them. Each thread has specific rules, so follow the thread's rules to get your map modded.
+*Warning: Do not spam requests in this channel, doing so may result in a [silence](/wiki/Silence). Post a mod request in `#modreqs` once every hour or so.*
```

Comment:
Although writing `Notice` here does seem to tone down the severity, I'm pretty sure this should be a warning since that's the purpose of this sentence.

(in the [ASC](https://github.com/ppy/osu-wiki/blob/master/wiki/Article_styling_criteria/Formatting/en.md#notice))

> `Warning` should be used to warn the reader that action may be taken against them.